core/chains/evm/txmgr/eth_resender.go

```diff
@@ -112,11 +117,25 @@ func (er *EthResender) resendUnconfirmed() error {
 		if err != nil {
 			return errors.Wrap(err, "failed to FindEthTxAttemptsRequiringResend")
 		}
+		oldestAttempt, exists := findOldestUnconfirmedAttempt(attempts)
+		if exists {
+			// Wait at least 2 times the EthTxResendAfterThreshold to log critical with an unconfirmedTxAlertDelay
+			if (time.Since(oldestAttempt.CreatedAt) > er.config.EthTxResendAfterThreshold()*2) &&
+				time.Since(er.lastAlertTimestamps[k]) >= unconfirmedTxAlertDelay {
```

Comment:
Can't lastAlertTimestamps[k] ever return nil if there's no entry beforehand?

Reply:
It does, so the first time `time.Since` is equal to `time.Now` and the condition holds true. This is reproduced in the `Test_EthResender_alertUnconfirmed` test case.